### DISCUSSION TITLE
Can't use 'tar -xzf' extract archive file

### DISCUSSION THREAD

**Discussion by DekelDevunet** [actions/checkout#1459] [Polls]
Wondering what currently is being done in the community regards the latest issue:
https://github.com/actions/checkout/issues/1448

**Comment by markrity** [comment]
Updated to v4, looked like it solved the issue , but after couple of runs fails also on v4.
So you can stay on v3 , since when you re-run it passes on v3. So there is no really reason to upgrade to v4

**Reply by DekelDevunet** [reply]
Thanks for sharing!
